virtcontainers/pkg/cloud-hypervisor/cloud-hypervisor.yaml

```diff
@@ -295,6 +295,12 @@ components:
         iommu:
```

Comment:
I am assuming this file is copied exactly from CLH  v0.5.0 (https://github.com/cloud-hypervisor/cloud-hypervisor/blob/v0.5.0/vmm/src/api/openapi/cloud-hypervisor.yaml). Can you please double check whether this file is correctly updated?

Local diff for your quick reference:
``` diff
*** /home/chenb/project/cloud-hypervisor/cloud-hypervisor/vmm/src/api/openapi/cloud-hypervisor.yaml     2020-02-03 15:10:44.712832743 -0800                                                                      
--- ./virtcontainers/pkg/cloud-hypervisor/cloud-hypervisor.yaml 2020-02-07 10:29:16.215954831 -0800                                                                                                              
***************
*** 277,288 ****
          iommu:                                                                                                                                                                                                 
            type: boolean                                                                                                                                                                                        
            default: false                                                                                                                                                                                       
-         num_queues:                                                                                                                                                                                            
-           type: integer                                                                                                                                                                                        
-           default: 1                                                                                                                                                                                           
-         queue_size:                                                                                                                                                                                            
-           type: integer                                                                                                                                                                                        
-           default: 128                                                                                                                                                                                         
                                                                                                                                                                                                                 
      NetConfig:                                                                                                                                                                                                 
        type: object                                                                                                                                                                                             
--- 277,282 ----
***************
*** 307,317 ****
          queue_size:                                                                                                                                                                                            
            type: integer                                                                                                                                                                                        
            default: 256                                                                                                                                                                                         
-         vhost_user:                                                                                                                                                                                            
-           type: boolean                                                                                                                                                                                        
-           default: false                                                                                                                                                                                       
-         vhost_socket:                                                                                                                                                                                          
-           type: string                                                                                                                                                                                         
                                                                                                                                                                                                                 
      RngConfig:                                                                                                                                                                                                 
        required:                                                                                                                                                                                                
--- 301,306 ----
```